invenio_app_ils/cli.py

```diff
@@ -79,6 +84,11 @@ def get_documents_items(internal_locations, n_docs, n_items):
             "title": "{}".format(lorem.sentence()),
             "authors": ["{}".format(lorem.sentence())],
             "abstracts": ["{}".format(lorem.text())],
+            "document_types": [random.choice(DOCUMENT_TYPES)],
```

Comment:
It's singular no? Is this relevant https://github.com/CERNDocumentServer/cds-dojson/blob/97abcdcffed04e695d69d15f0c42117d255cc407/cds_dojson/marc21/fields/books/base.py#L117 ?